src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier;
+
+    public Lottery() {
+        this.colorSupplier = new ColorSupplier();
+    }
+
+    public Ball getRandomBall() {
+        Color color = colorSupplier.getRandomColor();
+        int number = new Random().nextInt(100);
+        return new Ball(color, number);
+    }
```

Comment:
```suggestion
public class Lottery {
    private static final ColorSupplier colorSupplier = new ColorSupplier();
    private static final Random random = new Random();

    public Ball getRandomBall() {
        Color color = colorSupplier.getRandomColor();
        int number = random.nextInt(100);
        return new Ball(color, number);
    }
```
here better create like this